api/app/src/mappings/fitbit/body.ts

```diff
@@ -23,18 +23,22 @@ export const mapToBody = (
 
   if (fitbitUser) {
     if (fitbitUser.user.height) {
-      if (fitbitUser.user.heightUnit === METRIC) {
-        body.height_cm = convert(fitbitUser.user.height).from("in").to("cm");
+      if (fitbitUser.user.heightUnit === "en_US") {
```

Comment:
why did we remove the const?

Reply:
Because of the following:
We had `const METRIC = "en_US"`, but if you look at what Fitbit is returning for the units, they are different for height and weight:
`user.heightUnit = "en_US", user.weightUnit = "METRIC"`

So, because each unit was only used once, I thought it isn't necessary to use a const for it.

Reply:
Fair, but this is an external value that could be re-used elsewhere - especially if we'll be adding/changing logic. I'd just but both of them in a const.